_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,271 @@
+
+```html
+<style>
+	p {
+		font-size: 25px;
+	}
+</style>
+
+<p style="letter-spacing: 3px !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 4
+
+This `p` element has two [declared][] values for its `letter-spacing` property. The latest wins the [cascade sort][]. It has a value of `0.15em`, and is **wide enough**.
+
+```html
+<p style="letter-spacing: 0.1em !important; letter-spacing: 0.15em !important">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 5
+
+This `p` element has two [declared][] values for its `letter-spacing` property. The one which is [important][] wins the [cascade sort][]. It has a value of `0.15em`, and is **wide enough**.
+
+```html
+<p style="letter-spacing: 0.15em !important; letter-spacing: 0.1em">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 6
+
+This `p` element has two [declared][] values for its `letter-spacing` property. The one from the style sheet wins the [cascade sort][] because it is [important][]. Since it is not [declared][] via the `style` attribute, it matches the **cascade** condition. Note that neither the **wide enough**, nor the **not important** conditions are matched. Styles with [author origin][] declared by assistive technologies may override this style. This is nonetheless bad practice and sufficient spacing should be used.
+
+```html
+<style>
+	p {
+		letter-spacing: 0.1em !important;
+	}
+</style>
+
+<p style="letter-spacing: 0.15em">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Passed Example 7
+
+Both this `p` and `span` elements match the **not important** condition. For the `span`, the [cascaded][] value is `inherit !important`, thus the [computed][] value is the [inherited][] value, that is the [computed][] value of its parent, and it is not [important][]. Note that neither the **wide enough**, nor the **cascade** conditions are matched. Styles with [author origin][] declared by assistive technologies may override this style. This is nonetheless bad practice and sufficient spacing should be used.
+
+```html
+<p style="letter-spacing: 0.1em">
+	<span style="letter-spacing: inherit !important;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 8
+
+Both this `p` and `span` elements match the **not important** condition. For the `span`, the [cascaded][] value is `unset !important`, which is equivalent as `inherit` since it is an inherited property, thus the [computed][] value is the [inherited][] value, that is the [computed][] value of its parent, and it is not [important][]. Styles with [author origin][] declared by assistive technologies may override this style. This is nonetheless bad practice and sufficient spacing should be used.
```

Comment:
```suggestion
The [computed][] `letter-spacing` property of the `span` element is `unset`, which causes it to become inherited from the `p` element. Because the [computed][] `letter-spacing` property of the `p` element is not [important][], the `span` element has a [computed][] `letter-spacing` property that is not [important][], even though the [declared][] `letter-spacing` property is.
```